plugins/ubench-agent/src/main/java/org/renaissance/plugins/ubenchagent/Main.java

```diff
@@ -0,0 +1,68 @@
+package org.renaissance.plugins.ubenchagent;
+
+import org.renaissance.Plugin;
+
+import java.util.List;
+
+import cz.cuni.mff.d3s.perf.Measurement;
+import cz.cuni.mff.d3s.perf.BenchmarkResults;
+
+public class Main implements Plugin,
+    Plugin.BenchmarkSetUpListener,
+    Plugin.OperationSetUpListener,
+    Plugin.OperationTearDownListener,
+    Plugin.MeasurementResultPublisher {
+
+  final int eventSet;
+
+  public Main(String[] args) {
+    if (args.length != 1) {
+      warn("One argument should have been provided to the constructor.");
+      eventSet = -1;
+      return;
+    }
+    String[] events = args[0].split(",");
```

Comment:
The code here could be more benevolent, accepting events from multiple arguments :-)
It would allow more flexible formatting of the command line using the plugins.

Reply:
Right, fixed in c6acae8d3cce117ddc396078d0e7ea104e632a4a.